### DISCUSSION TITLE
How to use rainbowkit with @wagmi/core ?

### DISCUSSION THREAD

**Discussion by alienzhangyw** [rainbow-me/rainbowkit#1846] [Q&A] [answered]
I don't want to use the wagmi hooks, how can I get the connected account or signer without the hooks (for example, out of a react component)?
Is any examples to connect wallet using rainbowkit and then interact with smart contracts using @wagmi/core or viem or ethers.js?

**Comment by 0xmkh** [comment, marked as answer]
@alienzhangyw We don't use `@wagmi/core`, but we do use `wagmi` and `viem`. You could get connected account via `getAccount` function from `@wagmi/core`. Here is the link to the docs: https://wagmi.sh/core/api/actions/getAccount#getaccount
